ReportSimulationOutput/measure.rb

```diff
@@ -1448,19 +1433,33 @@ def check_for_errors(runner, outputs)
       return false
     end
 
+    # Check total vs net energy use
+    has_pv = @hpxml_bldgs.count { |hpxml_bldg| !hpxml_bldg.pv_systems.empty? } > 0
+    if has_pv
+      if @totals[TE::Total].annual_output == @totals[TE::Net].annual_output
+        runner.registerError('Building has PV but total/net energy uses are equal.')
```

Comment:
Could this be inadvertently thrown if, e.g., PV doesn't produce? (I'm not sure if that's actually possible.)

Reply:
Fair point. It's technically possible because a HPXML PV system is allowed to have an output power of zero. I just tried an HPXML with that but it failed because we create an EnergyPlus object w/ zero and E+ throws an error (so clearly no one has ever tried it). In any case, I'll fix that and then replace the above error-check with unit test asserts.

Reply:
Addressed in https://github.com/NREL/OpenStudio-HPXML/pull/1930/commits/140dfe3443bc095a4df90e527d0967926418f8e6.